sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryId.kt

```diff
@@ -0,0 +1,44 @@
+package io.sentry.kotlin.multiplatform
+
+class SentryId(private val sentryIdString: String) {
+    var uuidString: String = ""
+        private set
+
+    companion object {
+        val EMPTY_ID: SentryId = SentryId("00000000-0000-0000-0000-000000000000")
+    }
+
+    init {
+        if (sentryIdString.isEmpty()) {
+            uuidString = "00000000-0000-0000-0000-000000000000"
```

Comment:
This could call `EMPTY_ID.toString()` most likely

Reply:
Can we generate UUID on Kotlin stdlib? If not, using the `EMPTY_ID` is fine.

Reply:
I couldn't find a UUID in the stdlib but there's libraries like these ones: https://github.com/benasher44/uuid